ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -1044,6 +1044,33 @@ then
 	fi
 
 	#
+	# RC car setup
+	#
+	if [ $VEHICLE_TYPE == rccar ]
```

Comment:
They can (and should) both be the same thing. Either rover or ugv sound fine to me. The existing axialracing script can move into a vehicle init script. I can help with this if you're not sure.